src/tools/loggers/consoleLogger.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-console */
+// Since using `console` here is kind of the whole point.
+import { logLevels } from '@tvkitchen/base-constants'
+
+const log = (level, message) => {
+	const structuredMessage = JSON.stringify({
+		level,
+		message,
+	})
+	switch (level) {
```

Comment:
Another way to do this would be
```js
console[level === 'fatal' ? 'error' : level](structuredMessage)
```

Reply:
I think the "tradeoff" is the loss of default behavior (e.g. if someone ran `logger.log("PANTSPARTY", "wooooooo!"` it would error)...  but that's probably OK?

Reply:
Wait... I have an idea...

Reply:
Oh the other thing that shorthand approach does is rely on the (coincidental?) coupling of the log level constants being identifcal to the console method names (e.g. lower case).

I agree it feels verbose but I think that for now I'll just leave it as is and honestly we'll probably wind up deleting this whole thing later when we revisit logging...